.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+## **BEFORE YOU SUBMIT** please read the following:
+
+**I'm submitting a bug report**
+**I'm submitting a feature request**
+**I'm submitting a support request**
+
+
+**Webpack version:**
+1.10.x/2.x
+
+**Webpack Karma version**: 1.7.x
+
+
```

Comment:
You should add `karma` version as well, as that can have quite the impact sometimes.